assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/MoneyField/updateInputsSizes.tsx

```diff
@@ -0,0 +1,39 @@
+interface updatedPercentSignInputProps {
+	beforeWidth: number;
+	inputWidth: number;
+	signB4: boolean;
+}
+
+const updatedPercentSignInput = ({ beforeWidth, inputWidth, signB4 }: updatedPercentSignInputProps): void => {
+	document
+		.querySelectorAll('.ee-money-field--with-percent-sign .ee-input__price-field')
+		.forEach((element: HTMLElement) => {
+			if (signB4) {
+				element.style.marginLeft = `${beforeWidth}px`;
+			}
+
+			element.style.width = `${inputWidth}px`;
+		});
+};
+
+interface updateInputsSizesProps {
+	isPercent: boolean;
+	signB4: boolean;
+}
+
+const updateInputsSizes = ({ isPercent, signB4 }: updateInputsSizesProps): void => {
```

Comment:
May be just name it `updateInputSizes` without `s` after `Input`

Reply:
done, thx